packages/analytics/metric-cards/src/components/MetricCardContainer.vue

```diff
@@ -93,7 +97,7 @@ const formatCardValues = (card: MetricCardDef): MetricCardDisplayValue => {
     metricValue: card.formatValueFn ? card.formatValueFn(card.currentValue) : approxNum(card.currentValue, { capital: true, round: true }) || '0',
     metricChange: card.formatChangeFn ? card.formatChangeFn(change) : metricChange(change, props.hasTrendAccess, props.fallbackDisplayText),
     changePolarity: polarity,
-    icon: defineIcon(polarity, card.increaseIsBad),
+    trendIcon: defineIcon(polarity, card.increaseIsBad) as any,
```

Comment:
If you implement my other suggestion (providing a return type on the function) you should be able to get rid of the `any` here
```suggestion
    trendIcon: defineIcon(polarity, card.increaseIsBad),
```